### PR TITLE
ch(README.md): update the README file

### PR DESCRIPTION
**Pull request by FelixWambiri**
##### What does this PR do?
- Updates the README file with relevant information.
- Updates the README file with relevant links and images.
- Updates the SignIn page icons with the `Andelan` blue color.

##### Description of Task to be completed?
- Update the README file with the relevant information so as to give a new person in the project in-depth information.

##### Any background context you want to provide?
- The README had information about setting up but it was not detailed enough.

##### What are the relevant Pivotal Tracker stories?
N/A
##### Questions?
N/A
##### Screenshots?
image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,32 +3,83 @@
...
+1. Install [`Node JS`](https://nodejs.org/en/).
+2. To clone, run `git clone https://github.com/AndelaOSP/andela-societies-frontend.git`.
+3. `cd` into the root of the **project directory**.
+4. Install [`yarn`](https://yarnpkg.com/en/docs/install#mac-stable).
+5. Run `yarn install` on the terminal to install dependecies.
+6. Setup local development server.
+
+- In your terminal, execute the following command:
+  ```bash
+    sudo nano /etc/hosts
```
Comment (Kachulio1) [inline README.md]: maybe you could give an option of using `vim`

**Comment by daud1** [comment]
LGTM
Perhaps worth noting: there seems to be a typo in the PR description under the third heading.
I believe the line should read `...but it was not detailed enough.` instead.

File: tests/containers/forms/CommentsForm.test.jsx [hunk 2]
```diff
@@ -73,6 +73,13 @@ describe('<CommentsForm />', () => {
     expect(verifyRedemption).toHaveBeenCalled();
   });
 
+  it('should return null the default case when renderItemDetails is invoked with no selected Item', () => {
+    const instance = shallowWrapper.instance();
+    const selectedItem = {};
+    const result = instance.renderItemDetails(selectedItem);
+    expect(result).toBeNull();
+  });
+
```
Comment (kimani-ndegwa) [inline tests/containers/forms/CommentsForm.test.jsx]: Commented out code

Comment (FelixWambiri) [reply tests/containers/forms/CommentsForm.test.jsx]: I am going to delete the chunk of code, I think someone had mistakenly left it there.

**Comment by kimani-ndegwa** [comment]
Great job following best practice PR conventions. Also, @FelixWambiri as more of a small UI upgrade, please give the icons on this page match the blue on the Andela Logo.

**Comment by FelixWambiri** [comment]
Should I update the colour of the icons on this PR too?

File: tests/containers/forms/CommentsForm.test.jsx (hunk 2 above)
Comment (kimani-ndegwa) [reply tests/containers/forms/CommentsForm.test.jsx]:
```suggestion
Please remove these dead lines of code if we are not using them
```

Comment (kimani-ndegwa) [reply tests/containers/forms/CommentsForm.test.jsx]: Please remove the dead chunk of code if we are not using it

Comment (FelixWambiri) [reply tests/containers/forms/CommentsForm.test.jsx]: Thanks for the feedback, I have made the recommended changes.

**Pull request merged**